packages/components/src/utils/types.ts

```diff
@@ -1,15 +1,3 @@
-import { As } from 'reakit-utils/types';
-import { ViewOwnProps } from '@wp-g2/create-styles';
-
-export type Options<T extends As, P extends ViewOwnProps<{}, T>> = {
-	as: T;
-	name: string;
-	useHook: (props: P) => any;
-	memo?: boolean;
-};
-
-export type ResponsiveCSSValue<T> = Array<T | undefined> | T;
-
 export type SizeRangeDefault =
```

Comment:
Those types aren't consumed, should we remove all of them until they are necessary?

Reply:
Sure!